tests/scripts/pulp_rpm/test_rpm_sync_publish.sh

```diff
@@ -19,13 +19,32 @@ else
   curl_opt=""
 fi
 
+# Add content using JSON file
+cat <<EOT >> repo_config.json
+{
+"enabled":1,
+"repo_gpgcheck":1,
+"gpgcheck":1,
+"skip_if_unavailable":false,
+"assumeyes":true
+}
```

Comment:
what do you mean original?

Reply:
This is the input for the contents of config.repo. whether it will be generated or not will depend on boolean set on the distribution. Do you want me to also parse the generated config.repo when generate_config_repo is set to True?

Reply:
My question is: How does the file look in the published repository? Is it the same json, or does it look like ini-file?
And my thinking is it would be nice to the user to support both formats (or at least the one they are used to see).

Reply:
it's an ini file - it's a yum config file